src/components/slider/index.tsx

```diff
@@ -583,6 +592,7 @@ export default class Slider extends PureComponent<SliderProps, State> {
                 styles.track,
                 trackStyle,
                 styles.minimumTrack,
+                Constants.isRTL && this.props.disableRTL && styles.disableRTL,
```

Comment:
destruct the prop at the start with the rest of the props

Reply:
Done :)